Code/Mantid/docs/source/algorithms/ExportSampleLogsToCSVFile-v1.rst

```diff
@@ -9,37 +9,63 @@
 Description
 -----------
 
-Algorithm 'LoadSampleLogsToCSVFile' exports a specified set of sample logs
+Algorithm *LoadSampleLogsToCSVFile* exports a specified set of sample logs
 , which are stored in a MatrixWorkspace, to a CSV file.
+The header for the sample log csv file can also 
+be created by this algorithm in a seperate *header* file. 
+
+CSV File format
+===============
+
+Sample logs are written to a csv file.   
+A tab separates any two adjacent values. 
 
 Each entry of each exported sample log will be an individual entry in the
 output CSV file,
 except in the situation that two entries with time stamps within time tolerance.
 
+The output CSV file has 2+n columns, where n is the number of sample logs 
+to be exported. 
 
-Time Zone
----------
-
-The time stamps of sample logs are recorded as UTC time.
-Some users wants to see the exported sample log as the neutron facility's local time.
-So the input property 'TimeZone' is for this purpose.
+Here is the definition for the columns. 
 
+-  Column 0: Absolute time in second
+-  Column 1: Relative to first log entry's time
+-  Column 2 to (2 + n) - 1: log values in the order determined by input
+   *SampleLogNames*
```

Comment:
Maybe users would understand it better if you number the columns from `1` to `2+n`, instead of `0` to `2+n-1`? Just wondering.